src/ui/public/index_patterns/index_patterns.js

```diff
@@ -44,5 +45,8 @@ define(function (require) {
     self.patternToWildcard = Private(require('ui/index_patterns/_pattern_to_wildcard'));
     self.fieldFormats = Private(require('ui/registry/field_formats'));
     self.IndexPattern = IndexPattern;
-  });
+  }
+
+  module.service('indexPatterns', Private => Private(IndexPatternsProvider));
+  return IndexPatternsProvider;
 });
```

Comment:
What does the use of Private here do vs. the previous vanilla usage of module.service?